randovania/exporter/hints/joke_hints.py

```diff
@@ -25,6 +25,14 @@
     "We've been trying to contact you about your ship's extended warranty.",
     "The Scan Visor is useful for gathering information.",
     "Want to become famous? Buy follo... <Transmission blocked>",
+    "The Dark Suit protects against the toxic atmosphere of Dark Aether.",
+    "The Light Suit is fully resistant to the effects of Dark Aether.",
+    "Your current Chozo Battle Suit version is SA1-4468-VM6-P.",
+    "May the Light of Aether shine upon you!",
+    "Bored? Solve this: (11! - (-5*cos(pi)*10!) + (-1/(-10^-6)) - (((27^4)*7-7)/4) - (1.801665E6))",
```

Comment:
Missing period

Reply:
Missing period is intended.
Since Echoes doesnt support code blocks, a period can be confusing right in a math problem.

Reply:
I'm concerned if this reads decently in-game.

Reply:
![grafik](https://github.com/randovania/randovania/assets/38186597/fc14a653-d9ce-48c6-b2e2-f7caa5d50a6a)
From length it is fine, but unfortunately, the ^ and * characters don't get displayed well :(

Reply:
Yeah, I'd just skip this one.